verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -282,6 +283,71 @@ bool LinterConfiguration::operator==(const LinterConfiguration& config) const {
   return ActiveRuleIds() == config.ActiveRuleIds();
 }
 
+absl::Status LinterConfiguration::AppendFromFile(
+    absl::string_view config_filename) {
+  // Read local configuration file
+  std::string content;
+
+  const absl::Status config_read_status =
+      verible::file::GetContents(config_filename, &content);
+  if (config_read_status.ok()) {
+    RuleBundle local_rules_bundle;
+    std::string error;
+    if (local_rules_bundle.ParseConfiguration(content, '\n', &error)) {
```

Comment:
It feels like we have done the 'read config into string, then send it to ParseConfiguration' somewhere else already ? Maybe we should have RuleBundle implement ParseConfigurationFile()...
(might be something for a later refactoring; if so, add TODO)

Reply:
I think this is the exact code you remember, I just moved it to a different file